src/AdoNetCore.AseClient/AseTransaction.cs

```diff
@@ -63,6 +63,8 @@ public new AseConnection Connection
             }
         }
 
+        public AseConnection AseConnection => Connection;
```

Comment:
Consider memory leaks.

Reply:
I'm not sure I follow what you mean with this one.

`Connection` is already a public property, this is just exposing it under an extra name; and `Connection` is already doing an `if (_isDisposed)` check

Reply:
Fair point.